### PR TITLE
add progressInfo

### PR DESCRIPTION
**Pull request by yairkukielka**
- Add progressInfo parameter to `parseQuery`

- Bump version to 1.6.0

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -3,7 +3,8 @@ language: node_js
 # test on node.js versions
 node_js:
   - 8
-  - 9
+  - 10
+  - 12
```
Comment (noodle71) [inline .travis.yml:7]: Why this change?

Comment (yairkukielka) [reply .travis.yml:7]: check if CI still works on these node versions

**Review by noodle71 — APPROVED** [review]
ok!
